server/infrastructure/player_repository.py

```
@@ -83,3 +84,77 @@ async def get_player_by_id(self, player_id: int) -> Optional[Dict]:
+        try:
+            self.db.collection("teams").document(team_id.upper()).set(
+                {
+                    "roster_average": avg,
+                    "roster_average_updated_at": datetime.utcnow().isoformat() + "Z",
+                },
+                merge=True,
+            )
+        except Exception:
+            self._logger.exception("set_team_roster_average failed for %s", team_id)
+
+    def set_league_averages(self, league_doc: Dict[str, Any]) -> None:
```

Ruff (0.14.3)

Redefinition of unused `set_league_averages` from line 113

(F811)